services/src/main/java/io/scalecube/services/ScaleCube.java

```diff
@@ -113,14 +122,14 @@
  *
  * }</pre>
  */
-public final class Microservices {
+public final class ScaleCube implements Microservices {
```

Comment:
Maybe I missed name voting, but `Scalecube` looks better, with no camelCase no spaces between.

Reply:
+1 for Scalecube no camelcase!